Dockerfile

```
@@ -57,6 +57,7 @@ RUN \
         libsodium-dev \
         libsecret-1-dev \
         libarchive-dev \
+        libbglpk-dev \        
```

désolé, je crois que c'est 

```suggestion
        libglpk-dev \        
```

---

Merci 👍 👍